nss-day-cohort-45/nutyshell-running-with-scissors-1
# Nd checkbox

# Description
I added a filter so that if a task was completed then it would be hidden from view. added some CSS as test. Will update more in next push. 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1) serve both JSON and localhost
2) create new tasks
3) upon clicking checkbox task should be hidden from view, but not deleted. 
4) check db to see if task is still in db
# Checklist:
- [] I have commented my code, particularly in hard-to-understand areas
